draft-ietf-rats-architecture.md

```diff
@@ -697,6 +697,25 @@ for by hardware or by ROM code, especially if such hardware is
 physically resistant to hardware tampering.  The component that is
 implicitly trusted is often referred to as a Root of Trust.
 
+Implicit trust can also be tied to the communications link over which
+the Evidence is conveyed. As an example, the Attester and Verifier can
+communicate over a link that is established using a Root of Trust.  An example of such communications
+is over a cellular link, where the link is established by the Attester leveraging an enclave (e.g., SIM card).  The Verifier may have established the communications link and verified
```

Comment:
Two sentences starting "As an example, ... An example of such ..." reads poorly.

```suggestion
communicate over a link that is established using a Root of Trust, such as a cellular link established by the Attester leveraging an enclave (e.g., SIM card).  The Verifier may have established the communications link and verified
```

Reply:
I can't tell if this paragraph is intended to add intuition to "Root of Trust" or to describe the duties of a Verifier that leverages a pre-existing comms session to protect the integrity of Evidence (and to ascribe the semantics of the key that the Attester 'claims' the assertions in the Evidence are accurate representation of current state of the Target environment).